src/share/poudriere/jail.sh

```diff
@@ -64,6 +64,10 @@ Options:
                      building for a different TARGET ARCH than the host.
                      Only applies if TARGET_ARCH and HOST_ARCH are different.
                      Will only be used if -m is svn*.
+Options for -d:
```

Comment:
Are you familiar with manpages? It needs to be updated, but I can do it later too.